test/index.js

```diff
@@ -28,3 +34,25 @@ test('errors on too low version', async t => {
 	await t.throwsAsync(np('1.0.0', defaultOptions), /New version `1\.0\.0` should be higher than current version `\d+\.\d+\.\d+`/);
 	await t.throwsAsync(np('1.0.0-beta', defaultOptions), /New version `1\.0\.0-beta` should be higher than current version `\d+\.\d+\.\d+`/);
 });
+
+test('should not enable 2fa if the package is existed', async t => {
+	const getEnable2faArgsStub = sinon.stub().returns([]);
+	const np = proxyquire('../source', {
+		del: sinon.stub(),
+		'./prerequisite-tasks': sinon.stub(),
+		'./git-tasks': sinon.stub(),
+		'./npm/enable-2fa': {
+			getEnable2faArgs: getEnable2faArgsStub
+		}
+	});
+
+	await t.notThrowsAsync(np('1.0.0', {
+		...defaultOptions,
+		availability: {
+			isAvailable: true,
+			isUnknown: false
+		},
+		preview: true
+	}));
+	t.is(getEnable2faArgsStub.notCalled, true);
```

Comment:
It seems to me like this validates correct implementation rather than behavior. Maybe stub [`enable2fa`](https://github.com/sindresorhus/np/blob/5d1a00bb5c40c30f6adb30b6ad4719d841de5e6b/source/npm/enable-2fa.js#L17) instead of this method?

Reply:
@itaisteinherz Never mind previous changes because it was labeled with on-hold. Now it is ready of review.